### PR TITLE
Add showLogLevels

### PR DESCRIPTION
**Pull request by pbrisbin**
I have a use-case where I'd like to pass the current App's `LOG_LEVEL`
onto a sub-process that also uses Blammo. Once we've read it into a
`LogLevels`, there was no way to faithfully reproduce it for the child
process without such a function.

We could re-organize things so we retain and pass it through as an
opaque `Text`, but that would be non-trivial in this case. Adding this
function seems reasonable enough and means we won't have to.

### CODE REVIEW

File: tests/Blammo/Logging/LogSettings/LogLevelsSpec.hs [hunk 1]
```diff
@@ -44,7 +61,9 @@ spec = do
       shouldLogLevel ll2 "bar" LevelInfo `shouldBe` False
 
     it "can override multiple sources" $ do
-      let Right ll = readLogLevels "debug,Amazonka:warn,SQL:info"
+      let
+        ll =
+          newLogLevels LevelDebug [("Amazonka", LevelWarn), ("SQL", LevelInfo)]
```
Comment (pbrisbin) [inline tests/Blammo/Logging/LogSettings/LogLevelsSpec.hs:66]: This was a drive-by, but meant I could remove the `-Wno` pragma.

**Review by xave — APPROVED** [review]
